package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "spyro-server",
+  "version": "0.1.0",
+  "private": true,
+  "main": "index.js",
+  "scripts": {
+    "prettier": "prettier --write \"./**/*.js?(x)\" \"./**/*.md\"",
+    "lint": "eslint --ext .js,.jsx .",
+    "lint:fix": "eslint --fix --ext .js,.jsx .",
+    "start": "node ./bin/www",
+    "dev": "nodemon ./bin/www"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "dependencies": {
+    "axios": "^0.19.2",
+    "cookie-parser": "~1.4.4",
+    "debug": "~2.6.9",
+    "eslint": "^6.1.0",
+    "eslint-config-airbnb": "^18.0.1",
+    "eslint-config-prettier": "^6.3.0",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-react": "^7.14.3",
+    "eslint-plugin-react-hooks": "^1.7.0",
+    "express": "~4.16.1",
+    "http-errors": "~1.6.3",
```

Comment:
Not using CORS dependency as of now since It's working locally without any issues. If needed I'll add it once I'm done with testing after server is deployed.